src/test/ui/type-alias-impl-trait/multiple-def-uses-in-one-fn.stderr

```diff
@@ -0,0 +1,35 @@
+error: defining use generics `[B, A]` differ from previous defining use
+  --> $DIR/multiple-def-uses-in-one-fn.rs:10:45
+   |
+LL | fn f<A, B: 'static>(a: &'static A, b: B) -> (X<A, B>, X<B, A>) {
+   |                                             ^^^^^^^^^^^^^^^^^^
+   |
+note: previous defining use with different generics `[A, B]` found here
+  --> $DIR/multiple-def-uses-in-one-fn.rs:10:45
+   |
+LL | fn f<A, B: 'static>(a: &'static A, b: B) -> (X<A, B>, X<B, A>) {
+   |                                             ^^^^^^^^^^^^^^^^^^
```

Comment:
This is quite confusing to me :(

Reply:
I think the spans could be better here: the error should have a span of `X<B, A>` and the note should have a span of `X<A, B>`